pennylane_lightning/core/src/simulators/lightning_qubit/measurements/MeasurementsLQubit.hpp

```diff
@@ -125,12 +128,29 @@ class Measurements final
             }
             ind_probs++;
         }
-        // Transposing the probabilities tensor with the indices determined
-        // at the beginning.
+
+        // Permute the data according to the required wire ordering
         if (wires != sorted_wires) {
-            probabilities = Pennylane::Util::transpose_state_tensor(
-                probabilities, sorted_ind_wires);
+            static constexpr std::size_t CACHE_SIZE = 8;
```

Comment:
What's the reason behind this magic cache size? It's a little bit two low.

Reply:
It's fine for now. This is from JET, and used to ensure reasonable partition across the axes. Too high, and wasteful, too low, and not optimal, but functional. We can optimise on this later once the release is all sorted.